lib/iris/tests/system_test.py

```diff
@@ -22,7 +22,7 @@
 
 
 class SystemInitialTest(tests.IrisTest):
-    def system_test_supported_filetypes(self):
+    def test_supported_filetypes(self):
```

Comment:
`pytest` requires test names to start with "test" or it just won't discover them.